src/accordian/Accordion.tsx

```diff
@@ -0,0 +1,186 @@
+        let items = this.state.panelItems;
+        const {prevItemIndex} = this.state;
+        if (prevItemIndex != -1 && prevItemIndex === index) {
+            items[index].isOpen = false;
+            this.setState({prevItemIndex: -1});
+            items[prevItemIndex].content = this.getItemContent(prevItemIndex, items[prevItemIndex].title, true);
+            this.setState({panelItems: items});
+        } else if (items[index].isOpen === true) {
+            items[index].isOpen = false;
+            items[index].content = this.getItemContent(index, items[index].title, true);
+            this.setState({panelItems: items});
+        } else {
+            if (!accordionMultiPanel && prevItemIndex != -1) {
+                items[prevItemIndex].isOpen = false;
+                items[prevItemIndex].content = this.getItemContent(prevItemIndex, items[prevItemIndex].title, true);
+            }
+            items[index].isOpen = true;
+            items[index].content = this.getItemContent(index, items[index].title, false);
+            this.setState({prevItemIndex: index});
+            this.setState({panelItems: items});
+        }
+        this.accordionPanel();
+    };
+
+    private accordionPanel(): React.ReactElement {
+        return (
+            <div>
+                {this.state.panelItems.map((content: any) => {
+                    return (
+                        <div className={classNames([ClassNames.ACCORDION_PANEL])}>
+                            {content.content}
+                            {content.isOpen && content.isOpen === true ? (
+                                this.accordionContent(content)
+                            ) : (
+                                <div
+                                    className={classNames([ClassNames.ACCORDION_COLLAPSED_CONTENT])}
+                                    aria-hidden="true"
+                                />
+                            )}
+                        </div>
+                    );
+                })}
+            </div>
+        );
+    }
+
+    private accordionContent(content: any): React.ReactElement {
+        return (
+            <div className={ClassNames.ACCORDION_COLLAPSED_CONTENT} aria-hidden="false">
+                <div className={ClassNames.ACCORDION_TOGGLE_CONTENT} style={{display: "block"}}>
+                    <div className={ClassNames.ACCORDION_INNER_CONTENT}>
+                        <div className={ClassNames.ACCORDION_COMPONENT}>{content.component}</div>
+                    </div>
+                </div>
+            </div>
+        );
+    }
+
+    getItemContent = (index: any, title: any, isPrevious: boolean) => {
+        const {accordionMultiPanel} = this.props;
+        let panelClass = isPrevious
+            ? ClassNames.ACCORDION_PANEL_INNER
+            : classNames([ClassNames.ACCORDION_PANEL_INNER, ClassNames.ACCORDION_PANEL_OPEN]);
+        let expanded = isPrevious ? false : true;

```

Comment:
Suggestion-  you can use this way  -
expanded = !isPrevious

Reply:
addressed